code/modules/vending/snack.dm

```diff
@@ -5,21 +5,24 @@
 	product_ads = "The healthiest!;Award-winning chocolate bars!;Mmm! So good!;Oh my god it's so juicy!;Have a snack.;Snacks are good for you!;Have some more Getmore!;Best quality snacks straight from mars.;We love chocolate!;Try our new jerky!"
 	icon_state = "snack"
 	products = list(/obj/item/reagent_containers/food/snacks/candy = 6,
+					/obj/item/reagent_containers/food/snacks/chocolatebar = 6,
```

Comment:
You can use coins on the hot drinks vender for cocobars

Reply:
That seems really weird, the snack vendor literally advertises chocolate.